pkg/embeddedyamls/yamls.go

```diff
@@ -2783,6 +2783,7 @@ rules:
     resources:
       - pods
     verbs:
+      - delete
```

Comment:
This file is auto generated, you should look at the original file `config/rbac/submariner-diagnose/role.yaml`